style.css

```diff
@@ -0,0 +1,1171 @@
+
+.button:hover {
+  background-color: #FF4B37;
+  color: white;
+}
+
+.light-button:hover {
+  background-color: #BEBEBA;
+  color: #FF4B37;
+}
+
+.sub-button {
+  background-color: transparent;
+  color: white;
+  border: 2px solid white;
+  padding: 8px;
+}
+
+.sub-button:hover {
+  background-color: transparent;
+  border-color: #FF4B37;
+  color: white;
+}
+
+.light-sub-button:hover {
+  border-color: #BEBEBA;
+  color: #BEBEBA;
+}
+
+span {
+  color: #A7A9AC;
+}
+
+/*---------- header styling ----------*/
+
+header {
+  width: 100%;
+  height: auto;
+  position: relative;
+}
+
+.header-home {
+  background-color: white;
+}
+
+.header-logo {
+  margin: 35px 0;
+  width: 80%;
+}
+
+nav ul {
+  font-size: 0;
+  margin: 90px 0 0 0;
+}
+
+nav li {
+  font-size: 1.1rem;
+  font-weight: 600;
+  display: inline-block;
+  margin: 22px;
+}
+
+nav a {
+  color: #17406C;
+  font-size: 1.1rem;
+}
+
+/*---------- main styling ----------*/
+
+main {
+  width: 100%;
+  height: auto;
+}
+
+.main-home {
+  background-color: #17406C;
+  background: url(images/surf.png) no-repeat center top;
+  background-size: cover;
+}
+
+.main-contact {
+  background-color: white;
+}
+
+.get-contact-img {
+  width: 350px;
+  margin-top: 50px;
+  margin-left: 80px;
+}
+
+.phone-img {
+  width: 350px;
+  height: auto;
+  position: absolute;
+  bottom: -100%;
+  right: 15%;
+}
+
+.main-column {
+  width: 50%;
+}
+
+.wordle {
+  width: 65%;
+  margin: 70px 30px 30px 30px;
+}
+
+.graph {
+  width: 60%;
+  margin: 50px 25px 25px 25px;
+  padding-right: 80px;
+}
+
+.main-home-p {
+  color: white;
+  margin: 25px 25px 100px 25px;
+  text-align: justify;
+  background-color: rgba(20,39,54,0.5);
+  padding: 12px;
+  border-radius: 5px;
+}
+
+.contact-buttons {
+  margin: 70px auto 80px auto;
+}
+
+.main-round-corners {
+  border-top-left-radius: 2rem;
+  border-top-right-radius: 2rem;
+  background-color: #FF4B37;
+  max-width: 1100px;
+}
+
+/*---------- section styling ----------*/
+
+section {
+  width: 100%;
+  height: auto;
+}
+
+.section-home {
+  margin-top: 60px;
+}
+
+.section-home a {
+  font-weight: 600;
+  color: #142736;
+}
+
+.section-home a:hover {
+  color: #FF4B37;
+}
+
+.section-column-home {
+  width: 33.33%;
+  height: auto;
+  text-align: justify;
+}
+
+.section-column-contact {
+  width: 60%;
+  height: auto;
+}
+
+.section-mini-column {
+  width: 40%;
+  height: auto;
+  margin-top: 155px;
+  padding-left: 40px;
+}
+
+section p {
+  text-align: justify;
+  margin: 0 auto;
+  color: #222B30;
+}
+
+section h2 {
+  color: #222B30;
+  margin: 20px;
+}
+
+.section-p-home {
+  width: 90%;
+}
+
+.section-p-contact {
+  color: #BEBEBA;
+  width: 100%;
+  font-size: 1.05rem;
+  line-height: 1.5;
+}
+
+em {
+  font-weight: 500;
+  color: white;
+}
+
+.section-contact {
+  background-color: #222B30;
+  padding-top: 30px;
+  padding-bottom: 80px;
+}
+
+.borders {
+  border-right-style: solid;
+  border-left-style: solid;
+  border-width: 1px;
+  border-color: #A7A9AC;
+}
```

Comment:
I was about to say write this inline but I realized this is far more efficient. Good job.